webrtc.html

```diff
@@ -6112,6 +6406,75 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
         </tr>
       </tbody>
     </table>
+
+    <p>The following events fire on <code><a>RTCIceTransport</a></code>
+    objects:</p>
+
+    <table>
+      <tr>
+        <th>Event name</th>
+
+        <th>Interface</th>
+
+        <th>Fired when...</th>
+      </tr>
+
+      <tbody>
+        <tr>
+          <td><dfn id=
+          "event-icetransport-statechange"><code>statechange</code></dfn></td>
+
+          <td><code><a>Event</a></code></td>
+
+          <td>The <code><a>IceTransport</a></code> state changes.</td>
+        </tr>
+
+        <tr>
+          <td><dfn id=
+          "event-icetransport-gatheringstatechange"><code>gatheringstatechange</code></dfn></td>
+
+          <td><code><a>Event</a></code></td>
+
+          <td>The <code><a>IceTransport</a></code> gathering state
+          changes.</td>
+        </tr>
+
+        <tr>
+          <td><dfn id=
+          "event-icetransport-selectedcandidatepairchange"><code>selectedcandidatepairchange</code></dfn></td>
+
+          <td><code><a>Event</a></code></td>
+
+          <td>The <code><a>RTCIceTransport</a></code>'s selected
+            candidate pair</a> changes. </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <p>The following events fire on <code><a>RTCDtlsTransport</a></code>
+    objects:</p>
+
+    <table>
+      <tr>
+        <th>Event name</th>
+
+        <th>Interface</th>
+
+        <th>Fired when...</th>
+      </tr>
+
+      <tbody>
+        <tr>
+          <td><dfn id=
+          "event-dtlstransport-statechange"><code>statechange</code></dfn></td>
```

Comment:
likewise, you need to add an attribute `data-lt='dtlstransport-statechange'` otherwise respec will complains about duplicate definitions for `statechange`